AirCasting/Services/UpdateSessionParams.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  UpdateSessionParams.swift
+//  AirCasting
+//
+//  Created by Lunar on 01/04/2021.
+//
+
+import Foundation
+import CoreData
+
+class UpdateSessionParamsService {
+    
+    var context: NSManagedObjectContext {
+        PersistenceController.shared.container.viewContext
+    }
+    
+    func updateSessionsParams(session: Session, output: FixedSession.FixedMeasurementOutput) {
+        let dateFormatter = ISO8601DateFormatter.defaultLong
+    
+        session.id = Int64(output.id)
+        session.uuid = output.uuid
+        session.type = SessionType.from(string: output.type)?.rawValue ?? -1
```

Comment:
There's no such type. There has to be a value for session type and it's optional.  So if it's -1 there's something wrong in the app. I didn't want to make it mobile/fixed here, I think it might be more difficult to debug.